x-pack/plugins/ml/server/models/data_recognizer/modules/sample_datasets_flights/manifest.json

```diff
@@ -0,0 +1,30 @@
+{
+  "id": "sample_dataset_flights",
+  "title": "Kibana Sample Data Flights",
+  "description": "Find anomalies in Kibana Sample Data for Flights",
```

Comment:
I'd be more specific with the description - `Find anomalies in the sample data flight delays`.